at-quannguyen/W5/comment.rb

```diff
@@ -0,0 +1,18 @@
+class Comment < ApplicationRecord
+  after_save :increment_count_comment
+  belongs_to :post
+  belongs_to :user
+
+  def increment_count_comment
+    increment_count_post
+    increment_count_user
+  end
+
+  def increment_count_post
+    self.user.increment!(:comment_count)
+  end
+
+  def increment_count_user
+    self.post.increment!(:comment_count)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/b4590471b8c5c8b06f28a528a3e0e8c6dbb78b81/at-quannguyen/W5/comment.rb#L16"></span>Rails/SkipsModelValidations: Avoid using <code>increment!</code> because it skips validations. (http://guides.rubyonrails.org/active_record_validations.html#skipping-validations)</td>
    </tr>
  </tbody>
</table>